models/appModel.js

```diff
@@ -0,0 +1,40 @@
+var AppModel = Backbone.Model.extend({
+  constructor: function() {
+    this.videos = new VideoCollection();
+    this.currentVideo = new VideoModel();
+  },
+
+  getVideos: function(query) {
+    var self = this;
+    Backbone.ajax({
```

Comment:
Why didn't you use fetch?  This is like you're using a jQuery solution to a Backbone problem.  Let us know if you need to walk through how fetch works.  The downside is you did a lot of extra work that Backbone would do for you if you leaned on the fetch function.